packages/vuetify/src/components/VSpeedDial/VSpeedDial.js

```diff
@@ -63,10 +63,20 @@ export default {
     }
 
     if (this.isActive) {
+      let btnCount = 0
       children = (this.$slots.default || []).map((b, i) => {
-        b.key = i
-
-        return b
+        if (b.tag && b.componentOptions.Ctor.options.name === 'v-btn') {
+          btnCount++
+          return h('div', {
+            style: {
+              transitionDelay: btnCount * 0.05 + 's'
+            },
+            key: btnCount
```

Comment:
Shouldn't it be `key: i` here? Also there should be a possibility to provide keys by dev, not sure if it would work now

Reply:
thanks for review
fixed d805ed3